editors/sc-ide/CMakeLists.txt

```
@@ -52,7 +52,8 @@ if(APPLE)
         endforeach()
     endif()
 
-    foreach(file ${Sounds})
+    set(AdditionalStaticFiles ${Examples} ${Sounds})
++   foreach(file ${AdditionalStaticFiles})
```

```suggestion
    foreach(file ${AdditionalStaticFiles})
```
the `+` is not part of the CMake syntax but of the [git diff](https://www.atlassian.com/git/tutorials/saving-changes/git-diff) syntax :)